MicroProfile-Config/tck-runner/pom.xml

```diff
@@ -246,32 +246,34 @@
                 </testResources>
             </build>
         </profile>
-        
+
         <profile>
-            <id>mp-config-1.3</id>
+            <id>Payara Community</id>
             <activation>
                 <property>
                     <name>payara.version</name>
-                    <value>/4\..+|5\.19[1-4].*|5\.201.*/</value>
+                    <value>/^[5-9]\.[0-9]{4}(-SNAPSHOT|\..*)*/</value>
                 </property>
             </activation>
             <properties>
-                <microprofile.config.version>1.3</microprofile.config.version>
-                <microprofile.config.tck.version>1.3</microprofile.config.tck.version>
+                <microprofile.config.version>1.4</microprofile.config.version>
+                <microprofile.config.tck.version>1.4</microprofile.config.tck.version>
             </properties>
         </profile>
+
         <profile>
-            <id>mp-config-1.4</id>
+            <id>Payara Enterprise</id>
```

Comment:
While fine for Community, this probably needs more thought for Enterprise unless you want to take a "mañana" approach and cross that bridge when we come to it - It's entirely possibly that a single release of Enterprise will support two different versions of an API.

Reply:
For the time being I vote mañana to keep it clean while there's no difference between the two - plus, in my head if there were support for multiple versions of and API in one version of Enterprise, that activation probably wouldn't be based on Payara version anymore